jvm/src/main/scala/uk/gov/homeoffice/drt/protobuf/serialisation/FlightMessageConversion.scala

```diff
@@ -255,17 +251,34 @@ object FlightMessageConversion {
     Origin = PortCode(flightMessage.origin.getOrElse("")),
     PcpTime = flightMessage.pcpTime,
     Scheduled = flightMessage.scheduled.getOrElse(0L),
-    FeedSources = flightMessage.feedSources.flatMap(FeedSource(_)).toSet,
+    FeedSources = getFeedSource(flightMessage),
     CarrierScheduled = flightMessage.carrierScheduled,
-    ApiPax = flightMessage.apiPax,
     RedListPax = flightMessage.redListPax,
     ScheduledDeparture = flightMessage.scheduledDeparture,
-    TotalPax = flightMessage.totalPax.map(totalPaxSourceFromMessage).toMap
+    PassengerSources = getPassengerSources(flightMessage)
   )
 
-  def totalPaxSourceFromMessage(message: TotalPaxSourceMessage): (FeedSource, Option[Int]) = {
+  private def getPassengerSources(flightMessage: FlightMessage): Map[FeedSource, Passengers] = {
+    if (flightMessage.apiPax.isDefined) {
```

Comment:
Great to add API pax. We also need to add ActPax and TranPax in the case where we don't have these numbers persisted in the `PassengerSources` map. This needs some tests around it. Remember we have been persisting Arrival data for many years so we need to be able to deserialise it to the new arrival model accordingly